docs/070-project-management/mods-and-extentions.md

```diff
@@ -21,14 +22,8 @@ If your project requires a MOD or Extension, it is prudent to ensure everyone ag
 ## Common steps for a MOD or Extension:
 
 - Start with a short summary document which outlines the following:
-  1/ Why the change is required
-  2/ The current contractual language
-  3/ The proopsed contractual language
-  e.g.,
-  FROM
-  (old paragraphs, page numbers, etc.)
-  TO
-  (replacement language matched to old paragraphs, page numbers, etc.)
-
+  - Why the change is required
+  - The current contractual language
+  - The proposed contractual language (e.g., from <i>old paragraphs, page numbers, etc.<i> to <i>replacement language matched to old paragraphs, page numbers, etc.<i>
```

Comment:
@farooqzakhilwal can you try the emphasis markdown here instead of HTML? it would be:

```
(e.g., from _old paragraphs, page numbers, etc._ to _replacement language matched to old paragraphs, page numbers, etc._
```